.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Comment:
Don't you need to install the Bazel dependencies? unzip, g++ etc.

Reply:
so far it worked ^^